client/components/fields/editor/ContentTemplate.tsx

```diff
@@ -18,20 +19,65 @@ const mapStatToProps = (state) => ({
 });
 
 export class EditorFieldContentTemplateComponent extends React.PureComponent<IProps> {
-    render() {
-        const {gettext} = superdeskApi.localization;
-        const deskId = this.props.desk?._id;
-        const templates = deskId == null ?
+    get field(): string {
+        return this.props.field ?? 'content_template';
+    }
+
+    getAvailableTemplates(): Array<IContentTemplate> {
```

Comment:
I wouldn't mix the syntax, both methods are getters but each one defined differently, lets keep it consistent

Reply:
Good point, I'll fix that